src/web/core/src/main/java/org/geoserver/web/data/resource/DimensionEditor.java

```diff
@@ -464,15 +464,10 @@ else if (dimension.equals("time")) {
                     if (strategyModel.getObject() == Strategy.NEAREST) {
                         if (!DimensionDefaultValueSetting.TIME_CURRENT.equalsIgnoreCase(value
                                 .getValue())) {
-                            IValidationError err = new ValidationError("invalidNearestTimeReferenceValue " + Collections
-                                    .singletonMap("value", (Object) value.getValue()));
-                            value.error(err);
+                            value.error(new ValidationError("invalidNearestTimeReferenceValue"));
```

Comment:
In this case, the parameter was not used (hence removed).

DimensionEditor.referenceValue.invalidNearestTimeReferenceValue = Reference time value must be an ISO 8601 DateTime or word 'CURRENT'